src/index.ts

```diff
@@ -14,8 +14,31 @@ interface Options {
 	WithSummary: boolean; // Use to disable summary when running locally.
 }
 
+class RiffRaffUploadError extends Error {
+	constructor(message: string) {
+		super(message);
+		this.name = 'RiffRaffUploadError';
+	}
+}
```

Comment:
nit: not sure how much value subclassing `Error` is providing here, vs throwing a standard `Error`.